docs/docs/04-concepts/30-advanced/10-child-workflows.md

```diff
@@ -0,0 +1,3 @@
+# Child Workflows
+
+A Child Workflow allows you to 
```

Comment:
A Child Workflow allows you to launch a new WfRun that is tied to a Parent WfRun. This is similar to the concept of Child Process in operating systems, where a Parent Process spawns a Child Process and can pass down information to it at runtime.

Is this a good start to this page? If so I can continue, and if not, let's discuss this concept more.